week-9/Homework/mandatory/2-exercises/3-atm.js

```diff
@@ -12,12 +12,27 @@
  */
 
 class ATM {
-    // Add your code here
-
+    constructor () {
+        this.accountBalance = 100;
+        this.deposit = 0;
+        return this.accountBalance;
+    }   
+    make_deposit(deposit) {
+        this.deposit = deposit;
+        return this.deposit;
+    }
+    check_balance() {
+        this.accountBalance += this.deposit;
+        return this.accountBalance
+    }
+    make_withdrawl(withdraw) {
+        this.withdraw = withdraw;
```

Comment:
there is a small issue here, the idea is to have a balance and remove from it and add to it, no need to each return !